src/data/List.js

```diff
@@ -128,7 +128,7 @@ var List = function (dimensions, hostModel) {
         }
 
         var dimensionName = dimensionInfo.name;
-        dimensionInfo.type = dimensionInfo.type || 'float';
+        dimensionInfo.type = dimensionInfo.type || 'ordinal';
```

Comment:
I am afraid this default value should not be changed.
`List` is the core module which matters a lot to other features.
And It has no relationship with the logic of VisualMap.